agent_build/docker/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Final Scalyr Agent docker image that uses the previously built base image.
```

Comment:
If this file is auto generated from the base file, please add a comment which say auto-generated with instructions on how to generate it.

Reply:
No, that file is static and does not change. It just waits. that we specify the `BASE_IMAGE ` build arg to use it as a base image.